base/client.jl

```diff
@@ -211,6 +211,20 @@ end
 # call include() on a file, ignoring if not found
 include_ifexists(mod::Module, path::AbstractString) = isfile(path) && include(mod, path)
 
+function hascolorTTY()
```

Comment:
```suggestion
function ttyhascolor()
```
Minor nitpick, but this would be a more conventional name for this function.